language/en/acp/board.php

```diff
@@ -570,6 +570,7 @@
 	'SMTP_PORT'						=> 'SMTP server port',
 	'SMTP_PORT_EXPLAIN'				=> 'Only change this if you know your SMTP server is on a different port.',
 	'SMTP_SERVER'					=> 'SMTP server address',
+	'SMTP_SERVER_EXPLAIN'			=> 'Note that you have to provide the protocol that your server uses. If you are using SSL, this has to be "ssl://your.mailserver.com"',
```

Comment:
Wenn wir schon englische Originale an einer Stelle kommentieren an der sie tendenziell eher untergehen: Ich würde das eher ganz anders formulieren:
> Note that you have to provide the protocol you wish to use for the connection to your SMTP server. If you ...

Ansonsten finde ich nämlich auch nicht so ganz klar, welcher Server nun gemeint ist (schließlich läuft phpBB ja vielleicht auch auf einem) und die Frage des Bezugs hat sich erledigt, weil die einzige aktive Person "You" ist.